src/commissaire/containermgr/kubernetes/__init__.py

```diff
@@ -225,10 +226,12 @@ def register_node(self, name):
         resp = self._post(part, payload)
         if resp.status_code == 201:
             return True
```

Comment:
Not to be overly nit-picky, but you could eliminate the return value since it will never return `False`.  It either works or throws an exception.